src/zabapgit_repo_impl.prog.abap

```diff
@@ -533,7 +535,16 @@ CLASS lcl_repo IMPLEMENTATION.
 
     lt_cache = mt_local.
     lt_tadir = lcl_tadir=>read( get_package( ) ).
-    LOOP AT lt_tadir ASSIGNING <ls_tadir>.
+
+    LOOP AT it_filter ASSIGNING <ls_filter>.
```

Comment:
Is it possible to make `it_filter` (or probably `lt_filter` then) a sorted table, flip loop-read tables and use binary search ?